libcxx/docs/Status/Cxx23Issues.csv

```diff
@@ -46,7 +46,7 @@
 "`3473 <https://wg21.link/LWG3473>`__","Normative encouragement in non-normative note","November 2020","|Complete|","15.0","|format|"
 "`3474 <https://wg21.link/LWG3474>`__","Nesting ``join_views`` is broken because of CTAD","November 2020","|Complete|","15.0","|ranges|"
 "`3476 <https://wg21.link/LWG3476>`__","``thread`` and ``jthread`` constructors require that the parameters be move-constructible but never move construct the parameters","November 2020","",""
-"`3477 <https://wg21.link/LWG3477>`__","Simplify constraints for ``semiregular-box``","November 2020","","","|ranges|"
+"`3477 <https://wg21.link/LWG3477>`__","Simplify constraints for ``semiregular-box``","November 2020","|Nothing To Do|","","|ranges|"
```

Comment:
This should be `|Complete|` with the proper version number.
`Nothing To Do` means we really have nothing to do since the issue doesn't change the library. For example, https://cplusplus.github.io/LWG/issue3818 is a real `Nothing To Do`.

Reply:
I think this can be considered complete in 13.0 when _`copyable-box`_ got implemented (https://reviews.llvm.org/D102135, https://github.com/llvm/llvm-project/commit/6829db727e9e67dfdb70dd0846ffd4e48e00a98d). Although it's somehow curious that libc++ has never implemenented the original _`semiregular-box`_ before [P2325R3](https://wg21.link/p2325r3).

Reply:
Thanks, I've updated the status and version